public/css/tryAgain.css

```diff
@@ -0,0 +1,90 @@
+body {
+  direction: rtl;
+  margin: auto;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  background-color: grey;
+}
+
+@keyframes pop {
+	0% {
+		height: 0%;
+	}
+	50% {
+		height: 0%;
+	}
+	100% {
+		height: 100%;
+	}
+}
+
+@keyframes show-text {
+	0% {
+		opacity: 0;
+	}
+	50% {
+		opacity: 0;
+	}
+	100% {
+		opacity: 1;
+	}
+}
+
+.pop-up {
+	animation: pop 0.3s ease 1s;
```

Comment:
please ,, try to keep consistent with indentation ,, and choose if it is 4 spaces or 2 spaces

Reply:
ok , i always make beautify

Reply:
you should keep to the rules of eslint instead of using beautify